### ISSUE TITLE
In regards to your Open letter to the Linux community:  Are you goint to retract your paper from IEEE ?

### ISSUE THREAD

**Issue by evolvingfridge** [QiushiWu/qiushiwu.github.io#19]
After reading your [Open Latter](https://lore.kernel.org/lkml/CAK8KejpUVLxmqp026JY7x5GzHU2YJLPU8SzTZUNXU2OXC70ZQQ@mail.gmail.com/T/#u); you failed to mention the IEEE paper status, furthermore you failed to demonstrate understanding how such research can be conducted with community collaboration. 

To make it explicitly clear, this is not acceptable to use unethically gathered data by you in “hypocrite commit” publication.

Before the ban, per plonk email, complaint was made to your university and IEEE received at least one complaint; why are you refusing to retract you paper from IEEE ?

**Comment by aquast** [comment]
Agree, if the researchers regret is mentioned honestly, first thing is to retract the paper. Citing the results in the open letter makes me believe, they are not seriuosly apologizing.
